### PR TITLE
Allow 'connection.attempts' and 'connection.backoff.ms' to be configu…

### LINKED ISSUE #515: Kafka Connector stop if connection with database break with Sink Connector
Kafka Connector stop if connection with database break with Sink Connector. There should be retry in Sink connector as well. This is available in source connector

### PR DESCRIPTION
**Pull request by wongelz**
…red for sink connectors

Currently it's hard coded to JdbcSourceConnectorConfig defaults.

I believe this resolves #515 and #648

### CODE REVIEW

**Comment by wongelz** [comment]
[clabot:check]

**Comment by miguelpais** [comment]
Hi everyone, 
I'm really interested in having the functionality that @wongelz has proposed.

Could someone review/approve this Pull Request?

**Comment by k1th** [comment]
don't have write access, so this needs to go to @kkonstantine @gharris1727

**Comment by gopalakrishnan-subramani** [comment]
Can this pull request merge, this will solve numerous issues many customers going through?

File: src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkConfig.java [hunk 1]
```diff
@@ -81,6 +81,22 @@
   private static final String CONNECTION_PASSWORD_DOC = "JDBC connection password.";
   private static final String CONNECTION_PASSWORD_DISPLAY = "JDBC Password";
 
+  public static final String CONNECTION_ATTEMPTS =
+          JdbcSourceConnectorConfig.CONNECTION_ATTEMPTS_CONFIG;
+  private static final String CONNECTION_ATTEMPTS_DOC
+          = "Maximum number of attempts to retrieve a valid JDBC connection. "
+          + "Must be a positive integer.";
+  private static final String CONNECTION_ATTEMPTS_DISPLAY = "JDBC connection attempts";
+  public static final int CONNECTION_ATTEMPTS_DEFAULT = 3;
```
Comment (ncliang) [inline src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkConfig.java:90]: Maybe reuse the constants from `JdbcSourceConnectorConfig` for all of these, similar to the `QUOTE_SQL_IDENTIFIERS_CONFIG` family of constants?

**Review by ncliang — APPROVED** [review]
Thanks @wongelz . I think mostly LGTM with just minor comment. Once addressed, I can merge into `master` and it would be available with the next minor version bump release. I'm hesitant to backport this to existing releases for fear that users might have accidentally set these on sink.

File: src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkConfig.java (hunk 1 above)
Comment (wongelz) [reply src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkConfig.java]: Done!

**Pull request merged**